drivers/st/clk/stm32mp1_clk.c

```diff
@@ -0,0 +1,1611 @@
+/*
+ * Copyright (C) 2018, STMicroelectronics - All Rights Reserved
```

Comment:
This is related with my previous comment.

Reply:
The copyright and license headers are separate issues. For the disjunctively licensed files, I can see why you would like to keep the copyright header the same, i.e. so that the files are identical between U-Boot and TF-A. We're OK with that. But for the files that are unique to this project (and BSD-3-Clause only), we would prefer you to use the standard copyright header if possible.

Reply:
I will change the files in plat/st/stm32mp1 to use the standard TF-A copyright, except boot_api.h, which is the API coming from our ROM code. Several drivers copyright can also be changed, as those files are only use in TF-A. But I2C and PMIC drivers are shared with our HAL drivers (BSD license as well), so I should keep the ST copyright. And the dual license files, but the point is still under discussion on our side.